CMakeLists.txt

```diff
@@ -30,8 +30,8 @@ FetchContent_Declare(base-n
     GIT_REPOSITORY https://github.com/azawadzki/base-n.git
     GIT_TAG 7573e77c0b9b0e8a5fb63d96dbde212c921993b4)
 FetchContent_Declare(glslang
-    GIT_REPOSITORY https://github.com/KhronosGroup/glslang.git
-    GIT_TAG 12.3.1)
+    GIT_REPOSITORY https://github.com/BabylonJS/glslang.git
+    GIT_TAG main)
```

Comment:
We should not use `main`. This will cause the latest to be picked up. If we add additional code to `main`, it will get picked up immediately without testing. It would be better to use a commit id here since we don't have tags for our fork.